resources/js/src/app/components/itemList/filter/ItemRangeSlider.spec.ts

```diff
@@ -85,7 +88,7 @@ describe('ItemRangeSlider', () => {
         expect(UrlBuilder.updateSelectedFilters).toHaveBeenNthCalledWith(2, facet, 'price', { 'max': 140.55, 'min': 100 });
     });
 
-    // TODO: reenable the test once the tested functionality works properly
+    // // TODO: reenable the test once the tested functionality works properly
```

Comment:
I guess it can't be re-enabled already? Could you please try this test?

```suggestion
    // TODO: reenable the test once the tested functionality works properly
```